src/player.js

```diff
@@ -25,6 +27,16 @@ module.exports.getSlackUser = function(users, message) {
     if (message.match[1]) {
         nameOrId = message.match[1];
     }
+    var player = getSlackUserFromNameOrID(users, nameOrId);
```

Comment:
I think this can be written

`var player = getSlackUserFromNameOrID(users, nameOrId) || users.getByNameOrID(message.user)`